packages/web-app-search/src/portals/SearchBar.vue

```diff
@@ -129,6 +129,7 @@ export default defineComponent({
     const optionsDropRef = ref(null)
     const activePreviewIndex = ref(null)
     const term = ref('')
+    const loadFromRoute = ref(false)
```

Comment:
Small nitpick about the variable naming: I'd like to name this something like `restoreSearchFromRoute` to make it more descriptive.

Reply:
Fixed